app/views/contexts/section_summary_context.py

```diff
@@ -237,3 +328,159 @@ def _get_safe_page_title(self, title):
         return (
             safe_content(self._schema.get_single_string_value(title)) if title else ""
         )
+
+    @staticmethod
+    def _get_add_or_edit_blocks_primary(
+        primary_person_block: Mapping[str, Any]
+    ) -> tuple[str, str]:
+        primary_person_edit_block_id = primary_person_block["add_or_edit_block"]["id"]
+        edit_block_id = primary_person_block["add_or_edit_block"]["id"]
+
+        return primary_person_edit_block_id, edit_block_id
```

Comment:
What am I missing, this returns the same two ids ...?

Reply:
Yeah, it is just a duplicated code, removed it now and removed the method since it doesn't really do anything. Using multiple assignment for both blocks now.